libpages/server_test.go

```diff
@@ -0,0 +1,10 @@
+package libpages
+
+import "testing"
+
+// Copyright 2018 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+func TestStub(t *testing.T) {
```

Comment:
??

Reply:
I'm planning to add test for this package later, and wanted to add all tests to CI in this PR. Then I incorrectly thought there had to be test files for `go test` to exit with code 0; just realized no test files is fine too, so will just remove this file.